code-challenges.test.js

```diff
@@ -65,5 +149,43 @@ const vowelTester2 = "academy"
 const vowelTester3 = "challenges"
 // Expected output: 2
 
+describe("vowelSpotter", () => {
+    it("takes in a string and logs the index of the first vowel", () => {
+      const vowelTester1 = []
+      const vowelTester2 = []
+      const vowelTester3 = []
+      expect(vowelSpotter(vowelTester1))
+      expect(vowelSpotter(vowelTester2))
+      expect(vowelSpotter(vowelTester3))
+    })
+  })
+
+// Pseudo Code: 
+// create a variable array with vowels 
+// create a function called "vowelSpotter"
+// input a parameter called value
+// create a new variable for foundIndexes
+// create a for loop to iterate through index of the input value
+// now push the found vowel to foundIndexes
+// return the foundIndexes
 
 // b) Create the function that makes the test pass.
+const vowels = ['a', 'e', 'i', 'o', 'u'];
+
+const vowelSpotter = (value) => {
+  let foundIndexes = []
+  for(let i = 0; i < value.length ; i++) {
+    if (vowels.indexOf(value[i]) > -1){
+      foundIndexes.push(i);
+    }
+  } 
+  return foundIndexes.sort()[0]
+}
```

Comment:
Awesome, great job!  In this case, since you have index set to a variable already, there is no need to push it into an array and return the first index of that sorted array.  You can simply return i within your conditional.